src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -95,8 +95,38 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> CaptureClient::Capture(
   return capture_result;
 }
 
+static void SetupApiTableInfos(const ProcessData& process,
+                               const orbit_client_data::ModuleManager& module_manager,
+                               CaptureOptions* capture_options) {
+  size_t kMaxApiVersion = 0;
+  const char* kOrbitApiPrefix = "g_orbit_api_v";
+  for (const ModuleData* module_data : module_manager.GetAllModuleData()) {
+    std::optional<uint64_t> base_address = process.GetModuleBaseAddress(module_data->file_path());
+    if (!base_address.has_value()) {
+      ERROR("No base address found for module %s", module_data->file_path());
+      continue;
+    }
+
+    // TODO: The global variable symbols will be obtained in the next PR.
+    // module_data->GetDataSymbolsFromName(kOrbitApiPrefix)
+    std::vector<SymbolInfo*> dummy_global_variables_symbol_infos;
+
+    for (const SymbolInfo* symbol_info : dummy_global_variables_symbol_infos) {
+      for (size_t i = 0; i <= kMaxApiVersion; ++i) {
+        std::string orbit_api_table_name = absl::StrFormat("%s%u", kOrbitApiPrefix, i);
+        if (symbol_info->name() != orbit_api_table_name) continue;
+        ApiTableInfo* api_table_info = capture_options->add_api_table_infos();
+        api_table_info->set_module_path(module_data->file_path());
+        api_table_info->set_api_table_address(symbol_info->address() + base_address.value());
```

Comment:
Is the load bias missing here?

Reply:
yikes, good point, thanks. I think this is the proper way to use the load bias here?

```
api_table_info->set_api_table_address(symbol_info->address() + base_address.value() -
                                              module_data->load_bias());
```